README.md

```diff
@@ -53,7 +53,7 @@ We provide a Makefile to help you build Minideb locally. It should be run on a D
 $ sudo make
 ```
 
-To build an individual release (stretch, jessie or unstable)
+To build an individual release (stretch, buster or unstable)
```

Comment:
What an amazing work with the `qemu_build` script, it would be good to add some lines in the docs, would you mind including it in the PR?

Reply:
As it is for testing purposes in local users may need to know this script exists and how to use it ;)

Reply:
At the beginning this PR was not meant for merging so I did not update the docs, but given the interest around it, I'll sure write something down about the `qemu_build` script soon.

Reply:
Done. Tell me if the paragraph I added is sufficient or I have to add more details.

Reply:
Good enough to me! thanks!